src/Client.js

```diff
@@ -712,6 +712,18 @@ class Client extends EventEmitter {
         }, id);
     }
 
+    /**
+     * Returns a real id registered in whatsapp
+     * @param {string} id the whatsapp user's ID
+     * @returns {Promise<string>}
+     */
+    async getIdByNumber(id) {
+        return await this.pupPage.evaluate(async (id) => {
+            let result = await window.Store.Wap.queryExist(id);
+            return result.status === 200 ? result.jid._serialized : undefined;
```

Comment:
Hey,

Do you think it could be better if it returned null when you didn't find it?

Reply:
Yes, of sure. I will change it.

Reply:
Thanks by suggestion.